src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -17,9 +18,18 @@ def validate(self, data):
                 raise ValidationError(u'{} not a valid scope'.format(scope))
 
 
+class EventListField(serializers.WritableField):
+    def validate(self, data):
+        if not set(data).issubset(VALID_EVENTS):
+            raise ValidationError(u'Invalid event subscription: {}'.format(
```

Comment:
This failed for me when I tested it out :`{'events': [u'Invalid event subscription: issue']}`

`VALID_EVENTS = (issue.created,)` but then `EVENT_CHOICES = [['issue', 'Issue events']];`

Reply:
Everything should be consistent now. The API now accepts and exposes a list of "rolled up" event types (`issue`, etc.), then internally expands that to individual event types (`event.created`, `issue.created`, etc.)